pipeline/merger/src/main/scala/uk/ac/wellcome/platform/merger/services/Merger.scala

```diff
@@ -96,8 +96,21 @@ trait Merger extends MergerLogging {
           val redirects = redirectedSources.map(redirectSourceToTarget(target))
           logResult(result, redirects.toList, remaining.toList)
 
+          val redirectedIdentifiers =
+            redirectedSources.map { s =>
+              IdState.Identified(s.id, s.sourceIdentifier)
+            }.toSeq
+
+          val targetWork: Work.Visible[Identified] =
+            Work.Visible[Identified](
+              version = result.mergedTarget.version,
+              data = result.mergedTarget.data,
+              state = result.mergedTarget.state,
+              redirectSources = result.mergedTarget.redirectSources ++ redirectedIdentifiers
```

Comment:
Will this contain duplicates?

Reply:
I think it's theoretically possible, if the same target/source pair got merged with different rules?

I'm going to leave it as-is, because I don't think there's any problem with having duplicates here – this field shouldn't be displayed externally – and seeing lots of duplicates here might be a clue about weird merger behaviour.